packages/contracts/deployments/README.md

```diff
@@ -1,154 +1,105 @@
+-->
 
-### Predeploy contracts:
+*The contracts relevant for the majority of developers are `OVM_ETH` and the cross-domain messengers. The L2 addresses don't change.*
 
-| Contract                    | Address                                    |
-| --------------------------- | ------------------------------------------ |
-| OVM_L2ToL1MessagePasser     | 0x4200000000000000000000000000000000000000 |
-| OVM_L1MessageSender         | 0x4200000000000000000000000000000000000001 |
-| OVM_DeployerWhitelist       | 0x4200000000000000000000000000000000000002 |
-| OVM_ECDSAContractAccount    | 0x4200000000000000000000000000000000000003 |
-| OVM_SequencerEntrypoint     | 0x4200000000000000000000000000000000000005 |
-| OVM_ETH                     | 0x4200000000000000000000000000000000000006 |
-| OVM_L2CrossDomainMessenger  | 0x4200000000000000000000000000000000000007 |
-| Lib_AddressManager          | 0x4200000000000000000000000000000000000008 |
-| OVM_ProxyEOA                | 0x4200000000000000000000000000000000000009 |
-| OVM_ExecutionManagerWrapper | 0x420000000000000000000000000000000000000B |
-| OVM_GasPriceOracle          | 0x420000000000000000000000000000000000000F |
-| OVM_SequencerFeeVault       | 0x4200000000000000000000000000000000000011 |
-| OVM_L2StandardBridge        | 0x4200000000000000000000000000000000000010 |
-| ERC1820Registry             | 0x1820a4B7618BdE71Dce8cdc73aAB6C95905faD24 |
+### Predeploy contracts:
+|Contract|Address|
+|--|--|
+|OVM_L2ToL1MessagePasser|0x4200000000000000000000000000000000000000|
+|OVM_L1MessageSender|0x4200000000000000000000000000000000000001|
+|OVM_DeployerWhitelist|0x4200000000000000000000000000000000000002|
+|OVM_ETH|0x4200000000000000000000000000000000000006|
+|OVM_L2CrossDomainMessenger|0x4200000000000000000000000000000000000007|
+|OVM_GasPriceOracle|0x420000000000000000000000000000000000000F|
+|OVM_L2StandardBridge|0x4200000000000000000000000000000000000010|
+|OVM_SequencerFeeVault|0x4200000000000000000000000000000000000011|
+|OVM_L2StandardTokenFactory|0x4200000000000000000000000000000000000012|
+|OVM_L1BlockNumber|0x4200000000000000000000000000000000000013|
 
 ---
-
 ---
 
 ## LAYER 1
 
 ## OPTIMISTIC-KOVAN
 
-Network : **undefined (chain id: 69)**
-
-| Contract           | Address                                                                                                                         |
-| ------------------ | ------------------------------------------------------------------------------------------------------------------------------- |
-| OVM_GasPriceOracle | [0x038a8825A3C3B0c08d52Cc76E5E361953Cf6Dc76](https://undefined.etherscan.io/address/0x038a8825A3C3B0c08d52Cc76E5E361953Cf6Dc76) |
+Network : __undefined (chain id: 69)__
 
+|Contract|Address|
+|--|--|
+|OVM_GasPriceOracle|[0x038a8825A3C3B0c08d52Cc76E5E361953Cf6Dc76](https://undefined.etherscan.io/address/0x038a8825A3C3B0c08d52Cc76E5E361953Cf6Dc76)|
```

Comment:
We can delete this, it is a relic from when we had to deploy the gas oracle before it became a predeploy

Reply:
Requires deleting the `optimistic-kovan` deployments folder